recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -1,11 +1,12 @@
 from conan import ConanFile
+from conan.tools.meson import MesonToolchain, Meson
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.layout import basic_layout
 from conan.tools import files, scm, microsoft
```

Comment:
Should probably import functions directly from the various `conan.tools.x` modules.

Reply:
It's fine, it's documented that both are write: https://docs.conan.io/en/latest/reference/conanfile/tools.html

I personally prefer importing only the method, but importing its module it's allowed too.

Reply:
Then nevermind!